robot_specific_config/abb_irb4600_support/urdf/irb4600_40_255_macro.xacro

```diff
@@ -73,6 +88,9 @@
         </geometry>
         <xacro:material_abb_graphite_white/>
       </visual>
+      <xacro:cylinder_inertial radius="0.08" length="1.03" mass="13.95">
+        <origin xyz="0.8 0 0" rpy="0 ${80} 0" />
```

Comment:
Why `80`, and why is it in a `xacro` expression (the `{}`)?

Reply:
It should've been 90 I fixed it. Btw having the expression just like that without `radians`  worked. Look at this image (when it was 80degrees)

![MicrosoftTeams-image](https://github.com/sam-xl/abb_ros2/assets/104774720/9ff7b4c8-69bf-448c-987c-4cbd74c95939)